.github/workflows/release.yml

```diff
@@ -40,13 +42,24 @@ jobs:
             This is a automated release completed by GitHub Actions.
           draft: false
           prerelease: false
-      - name: Upload Release Asset
-        id: upload-release-asset
+      - name: Upload EXE
+        id: upload-client
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{steps.create_release.outputs.upload_url}}
           asset_path: dist/client.exe
           asset_name: client.exe
```

Comment:
Is there any reason it is compiling to client.exe? Just because that is the root of the Python name and that is how the compile builds? Can we redirect that? Would prefer to see the runnable exe as freescribe-client.exe - even fs-client would do.

Reply:
https://github.com/ClinicianFOCUS/FreeScribe/pull/3/commits/cb9beef55ae94cc1168b970784de12ccba595786 - updated pyinstaller

https://github.com/ClinicianFOCUS/FreeScribe/pull/3/commits/b5811cfce7de46ba2a8b7b9c15b0655bbfd5eff3 - changed name in release